Nos/Views/NIP05View.swift

```diff
@@ -43,13 +43,17 @@ struct NIP05View: View {
                             username: nip05Identifier,
                             publicKey: publicKey
                         )
+                    } catch URLError.cancelled {
+                        // Cancelled requests are common in bad networks. We
+                        // should keep things as they were to avoid reproducing
+                        // https://github.com/planetary-social/nos/issues/1161
+                        return
                     } catch {
                         isVerified = false
-                        Log.debug(error.localizedDescription)
-                    }
-                    withAnimation {
-                        self.verifiedNip05Identifier = isVerified
+                        let message = error.localizedDescription
+                        Log.debug("Error while verifying a NIP-05: \(message)")
```

Comment:
It always looks a little strange to me to see `Log.debug("Error`. If it's an error, perhaps we should use `Log.error`?

Reply:
It's an error but high chances it's not error level, but debug level. The reason is that it's a http request to external servers so this will throw errors for servers we don't even own or because the users doesn't have Internet connection (very normal). We would want to see this information only when debugging, in a normal scenario we shouldn't bother because it's expected to fail sometimes.

In order to do this 100% correct, we would need to differentiate real errors vs expected errors (like no Internet) and use debug and error levels accordingly. I'm not sure it worths our effort though.

Reply:
If you are still not convinced, let me know.

Reply:
That makes sense. I'm fine with leaving this.